ros2doctor/ros2doctor/command/doctor.py

```diff
@@ -48,13 +52,13 @@ def main(self, *, parser, args):
 
         # `ros2 doctor -r`
         if args.report:
-            all_reports = generate_reports()
+            all_reports = generate_reports(exclude_packages=args.exclude_packages)
             for report_obj in all_reports:
                 format_print(report_obj)
             return
 
         # `ros2 doctor
-        fail_category, fail, total = run_checks(include_warnings=args.include_warnings)
+        fail_category, fail, total = run_checks(include_warnings=args.include_warnings, exclude_packages=args.exclude_packages)
```

Comment:
there is flake8 error, https://build.ros2.org/job/Rpr__ros2cli__ubuntu_noble_amd64/47/testReport/ros2doctor.ros2doctor.test/test_flake8/test_flake8/. can you address it?

Reply:
Yeah, I saw it on the previous test, but wasn't sure if it was an error or a warning. I will have no choice than to create a local variable to reduce the length of the line. Will do it asap